gluetool/help.py

```diff
@@ -324,3 +326,129 @@ def functions_help(functions):
     {{ body }}
     {% endfor %}
     """)).render(FUNCTIONS=[function_help(func, name=name) for name, func in functions])
+
+
+def extract_eval_context_info(source, logger=None):
+    """
+    Extract information of evaluation context content from the ``source`` - a module
+    or any other object with ``eval_context`` property. The information we're looking
+    for is represented by an assignment to special variable, ``__content__``, in the
+    body of ``eval_context`` getter. ``__content__`` is expected to be assigned
+    a dictionary, listing context variables (keys) and their descriptions (values).
+
+    If it's not possible to find such information, or an exception is raised, the function
+    returns an empty dictionary.
+
+    :param gluetool.glue.Configurable source: object to extract information from.
+    :rtype: dict(str, str)
+    """
+
+    logger = logger or Logging.get_logger()
+
+    logger.debug("extract eval cotext info from '{}'".format(source.name))
+
+    # Cannot do "source.eval_context" because we'd get the value of property, which
+    # is usualy a dict. We cannot let it evaluate and return the value, therefore
+    # we must get it via its parent class.
+    eval_context = source.__class__.eval_context
+
+    # this is not a cyclic import, yet pylint thinks so :/
+    # pylint: disable=cyclic-import
+    from .glue import Configurable
+
+    # If eval context is the same as Configurable's - which is the original class for all modules
+    # and even gluetool.glue.Glue - it means its inherited from Configurable, therefore
+    # ``source`` does not have its own, and in that case we should return an empty string - module
```

Comment:
Which empty string do you mean? You are returning empty dictionary

Reply:
Right, fixed.